tests/test_command_copilot.py

```diff
@@ -315,15 +314,28 @@ def test_env_addons_parameters_file_with_different_addon_types(
 
         assert result.exit_code == 0
         validate_version.assert_called_once()
-        if addon_type == "s3":
-            assert (
-                "File copilot/environments/addons/addons.parameters.yml" not in result.output
-            ), f"addons.parameters.yml should not be included for {addon_type}"
-        else:
-            assert (
-                "File copilot/environments/addons/addons.parameters.yml overwritten"
-                in result.output
-            ), f"addons.parameters.yml should be included for {addon_type}"
+        assert (
+            "File copilot/environments/addons/addons.parameters.yml created" in result.output
+        ), f"addons.parameters.yml should be included for {addon_type}"
+
+    @patch("dbt_copilot_helper.jinja2_tags.version", new=Mock(return_value="v0.1-TEST"))
+    def test_env_addons_parameters_file_should_not_be_included_for_s3(
+        self, fakefs, validate_version
+    ):
+        fakefs.create_file(
+            ADDON_CONFIG_FILENAME,
+            contents=(S3_STORAGE_CONTENTS),
+        )
+        fakefs.create_file("copilot/web/manifest.yml")
+        fakefs.create_file("copilot/environments/development/manifest.yml")
```

Comment:
I think there may be scope for extracting these create_file setup steps to a helper method. It could improve readability of the tests.